src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -13,6 +13,9 @@
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
 
+
```

Comment:
whoa now

Reply:
I was trying to get checkstyle to call me out on it :c IDK how to make it work~

Reply:
ahhh I see

Reply:
I think it works properly now :3 it tags and comments every style error just the way id expect it to.